src/shared/ml-thirdparty-simulator/factories/transfer.ts

```diff
@@ -28,21 +28,24 @@ import * as faker from 'faker'
 import {
   AuthorizationsPutIdRequest,
   TransferIDPutRequest,
-} from '~/shared/ml-thirdparty-client/models/openapi';
+} from '~/shared/ml-thirdparty-client/models/openapi'
 
-import { TransferState } from '~/shared/ml-thirdparty-client/models/core';
+import { TransferState } from '~/shared/ml-thirdparty-client/models/core'
 
 export class TransferFactory {
   /**
    * Creates a `PUT /transfers/{ID}` request body that is normally sent
    * by Mojaloop as a callback to inform about the transfer result.
-   * 
+   *
    * @param _               transaction request id of the corresponsing authorization.
    * @param __              an authorization object as defined by the Mojaloop API.
    * @param transactionId   transaction id to be associated with the transfer object.
    */
-  public static createTransferIdPutRequest(_: string,
-    __: AuthorizationsPutIdRequest, transactionId: string): TransferIDPutRequest {
+  public static createTransferIdPutRequest(
+    _: string,
+    __: AuthorizationsPutIdRequest,
```

Comment:
lovely magic of underscores.... do you know that the floors in debugging hell are full of underscores?

Reply:
hi pawel - can you raise an issue for this - i just made style changes here and for the sake of separation of concerns I don't want to modify code not related to this ticket/PR

Reply:
https://github.com/mojaloop/pisp-demo-server/issues/26